src/SLCore.IntegrationTests/FileAnalysisTests.cs

```diff
@@ -28,41 +28,91 @@
 using SonarLint.VisualStudio.SLCore.Listener.Files.Models;
 using SonarLint.VisualStudio.SLCore.Listeners.Implementation;
 using SonarLint.VisualStudio.SLCore.Service.Analysis;
+using SonarLint.VisualStudio.SLCore.Service.Rules;
+using SonarLint.VisualStudio.SLCore.Service.Rules.Models;
 using SonarLint.VisualStudio.SLCore.State;
 
 namespace SonarLint.VisualStudio.SLCore.IntegrationTests;
 
 [TestClass]
 public class FileAnalysisTests
```

Comment:
There's a special test case that's missing: you can set the initial rule configuration via `InitializeParams.standaloneRuleConfigByKey`

Reply:
As discussed, I updated the following ticket to add the test case: https://sonarsource.atlassian.net/browse/SLVS-1288?atlOrigin=eyJpIjoiZTAyODJlNWIyY2Q3NGY0MThmMjg2N2QxOWMxNzNjNzEiLCJwIjoiaiJ9